documentation/book/proc-configuring-kafka-bridge-tls.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-tls.adoc
+
+[id='proc-configuring-kafka-bridge-tls-{context}']
+= Configuring TLS in Kafka Bridge
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator
+* If they exist, the name of the `Secret` for the certificate used for TLS Server Authentication, and the key under which the certificate is stored in the `Secret`
+
+.Procedure
+
+. (Optional) If they do not already exist, prepare the TLS certificate used in authentication in a file and create a `Secret`.
```

Comment:
We should mention here that secretcreated by the CO for the Kafka cluster can be used directly.

Reply:
NOTE: The secrets created by the Cluster Operator for Kafka cluster may be used directly.